ql/termstructures/volatility/equityfx/fixedlocalvolsurface.cpp

```diff
@@ -132,7 +132,8 @@ namespace QuantLib {
 
         for (Size i=0; i < strikes_.size(); ++i)
             for (Size j=1; j<strikes_[i]->size(); j++) {
-                QL_REQUIRE(strikes_[i]->at(j)>=strikes_[i]->at(j-1),
+                QL_REQUIRE(strikes_[i]->at(j)>=strikes_[i]->at(j-1)
```

Comment:
While we're looking at this: we're looping between 1 and the size, so the extra access check with `at` is not necessary here.  We can use `(*strikes[i])[j]` instead.  Or if the syntax is awkward, we can take a const reference to `*strikes[i]` in the outer loop and use that one in the inner loop...